war/src/main/less/base/layout-commons.less

```diff
@@ -3,13 +3,20 @@
 html {
   position: relative;
   min-height: 100%;
+  box-sizing: border-box;
 }
 
 body {
   margin: 0;
   padding: 0 0 40px 0;
 }
 
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
```

Comment:
This has changed from border-box to inherit, is that fine?

Reply:
Yes, it will take the value from the html tag